vk_video_decoder/include/vkvideo_parser/VulkanVideoParser.h

```diff
@@ -35,7 +35,7 @@ class IVulkanVideoDecoderHandler : public VkVideoRefCountBase {
                                       const uint8_t* pInitializeBufferMemory,
                                       size_t initializeBufferMemorySize,
                                       VkSharedBaseObj<VulkanBitstreamBuffer>& bitstreamBuffer) = 0;
-
+    virtual bool IsDstDpbDistinctImages() = 0;
```

Comment:
I don;t think we need this additional function - please see my comments about DPB/output images.